src/main/java/pl/gispartner/ResourceManagerApp/api/ResourceController.java

```diff
@@ -0,0 +1,25 @@
+package pl.gispartner.ResourceManagerApp.api;
+
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import pl.gispartner.ResourceManagerApp.model.ResourceEntity;
+import pl.gispartner.ResourceManagerApp.service.ResourceService;
+
+@RestController
+@RequestMapping("/api/resource")
+@RequiredArgsConstructor
+public class ResourceController {
+
+    private final ResourceService resourceService;
+
+    @PostMapping("/save")
+    public void saveResource (@RequestBody ResourceEntity resourceEntity){
```

Comment:
1. moznaby tu zwrocic id zapisanego obiektu, aby front mogl wykonywac operacja przy uzyciu identyfikatora zasobu
2. nie powinienes tworzyc sciezki urla (patha) jesli jest to usuwanie, tworzenie, edycja, odczyt - eng. CRUD (create, read, update, delete), od tego masz metody ktore Ci to charakteryzuja (POST, GET, PUT, DELETE)